src/api/mutations/account/useCreateAccountMutation.ts

```diff
@@ -1,26 +1,20 @@
-import { useMutation, useQueryClient } from '@tanstack/react-query';
-import toast from 'react-hot-toast';
-import { useNavigate } from 'react-router-dom';
+import { useMutation } from '@tanstack/react-query';
 
 import httpClient from '@src/api/httpClient';
 import { ENDPOINTS } from '@src/config/endpoints.config';
 import { QUERY_KEYS } from '@src/config/react-query.config';
-import { ROUTES } from '@src/config/routes.config';
 import { Profile } from '@src/types/profile.types';
 
 function useCreateAccountMutation() {
-  const navigate = useNavigate();
-  const queryClient = useQueryClient();
-
   return useMutation({
     mutationKey: [QUERY_KEYS.mutation.createAccount],
     mutationFn: async (profileToSave: Profile) => {
```

Comment:
i think argument name 'profile' it's enough to understand the goal profileToSave=>profile